pravega/schema-registry
# Issue 152: Fixing DISABLE_BASIC_AUTH

Signed-off-by: SrishT <Srishti.Thakkar@dell.com>

**Change log description**  
- Basic auth is not getting disabled when the setting DISABLE_BASIC_AUTHENTICATION is set to true.
- log.level is not getting set since the config property is not getting set using a corresponding environment variable.

**Purpose of the change**  
Fixes #152 

**What the code does**  
- Disables initialization and registration of the "Basic" Auth Handler if DISABLE_BASIC_AUTHENTICATION is set to true.
- Adds a field for including `log.level` as a config property
- Fixes inconsistencies in the helm charts

**How to verify it**  
- Check that the log level is getting reflected correctly in the logback.xml file
- UserPasswordFile and PasswordAuthSecret should not be required when basic auth is disabled
- Try deploying the schema registry charts with authentication enabled and ensure that the fields AUTHORIZATION_DOMAIN_RESOURCE_IDENTIFIER, AUTHORIZATION_NAMESPACE_RESOURCE_IDENTIFIER and AUTHORIZATION_GROUP_RESOURCE_IDENTIFIER are correctly reflected in the configmap when set in the values.yaml
